cms/djangoapps/contentstore/management/commands/tests/test_git_export.py

```diff
@@ -74,27 +74,15 @@ def test_error_output(self):
         """
         Verify that error output is actually resolved as the correct string
         """
-        output = StringIO.StringIO()
         with self.assertRaisesRegexp(CommandError, unicode(GitExportError.BAD_COURSE)):
             call_command(
-                'git_export', 'foo/bar:baz', 'silly',
-                stdout=output, stderr=output
+                'git_export', 'foo/bar:baz', 'silly'
             )
-        self.assertIn('Bad course location provided', output.getvalue())
-        output.close()
```

Comment:
@macdiesel the strings are already asserted in assertRaisesRegexp which is `unicode(GitExportError.BAD_COURSE)`. I'd only remove the `StringIO` related stuff as the CommandError exceptions are no longer written to `stdout`.